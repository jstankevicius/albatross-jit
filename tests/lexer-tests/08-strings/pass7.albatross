"this   string  has actual  tabs"
# each space in the string above is a tab
# and here's some strings with just white-space, a mix of spaces and tabs
" "
"   "
"                                                       "
