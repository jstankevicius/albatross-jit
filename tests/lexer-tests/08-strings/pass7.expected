1 1 STRING 31 this   string  has actual  tabs
1 4 STRING 1  
1 5 STRING 3    
1 6 STRING 55                                                        
